framework/src/materials/DerivativeSumMaterial.C

```diff
@@ -38,6 +38,8 @@ DerivativeSumMaterial::validParams()
   // Advanced arguments to construct a sum of the form \f$ c+\gamma\sum_iF_i \f$
   params.addParam<std::vector<Real>>("prefactor", "Prefactor to multiply the sum term with.");
   params.addParam<Real>("constant", 0.0, "Constant to be added to the prefactor multiplied sum.");
+
+  params.addParam<bool>("validate_coupling", true, "Flag to validate coupling.");
```

Comment:
```suggestion
  params.addParam<bool>("validate_coupling", true, "Check if all variables the specified materials depend on are listed in the `args` parameter.");
```

That would make it a bit more descriptive